IncSet.c

```diff
@@ -12,6 +12,8 @@ in the source distribution for its full text.
 #include <ctype.h>
 #include <string.h>
 #include <stdlib.h>
+#include <limits.h>
```

Comment:
The include lines should be ordered alphabetically for headers in the same group (standard libc headers in this case).